src/utils/fetchClient.ts

```diff
@@ -29,7 +29,15 @@ function request<T>(
   // for a demo purpose we emulate a delay to see if Loaders work
```

Comment:
Consider using a more descriptive variable name instead of 'isLoad'. According to the checklist, it should be 'isLoading' or 'isLoaded'.